_data/contributors.yml

```
@@ -42,6 +40,53 @@
   packages-submitted: [""]
   packages-reviewed: ["pandera"]
   packages-editor: ["sevivi"]
+- name: Filipe Fernandes
+  advisory: true
+  bio: ''
+  organization: "IOOS, Conda Forge"
+  github_username: ocefpaf
+  github_image_id: 950575
+  contributor_type:
+    - contributor
+  packages-submitted: ["errdapy"]
+  packages-reviewed: [""]
+  packages-editor: [""]
+- name: Lindsey Heagy
+  advisory: true
+  bio: ''
+  organization: ""
+  github_username: lheagy
+  github_image_id: 6361812
+  contributor_type:
+    - contributor
+  packages-submitted: [""]
+  packages-reviewed: [""]
+  packages-editor: [""]
+- name: Martin Fleischmann
+  advisory: true
+  bio: 'Urban Data Scientist'
+  organization: "University of Strathclyde"
+  github_username: martinfleis
+  github_image_id: 36797143
```

```suggestion
  github_image_id: 36797143
  twitter: martinfleis
  mastodon: martinfleis@mastodon.social
  orcidid: 0000-0003-3319-3366
```

I hope this is the correct syntax. Please check @lwasser.